### ISSUE TITLE
Request: Visual Studio 2022 Support Please. 😁

### ISSUE THREAD

**Issue by Mike-E-angelo** [omsharp/LinePress#12]
Please consider supporting [Visual Studio 2022](https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022) with this great extension! 👍

**Comment by mayureshs** [comment]
Any updates on availability for VS 2022?

**Comment by smimon** [comment]
Seconded, VS2022 is out now and I would love to use this

**Comment by Mike-E-angelo** [comment]
As it's been 2 years since any activity here, we might consider creating a fork and introducing an `.Unofficial` extension in the marketplace.

**Comment by mayureshs** [comment]
VS 2022 already has a setting in Text Editor options

**Comment by vincentkosciuszko** [comment]
> y

wow thanks

**Comment by ghost** [comment]
> VS 2022 already has a setting in Text Editor options

wow thanks @mayureshs

**Comment by stkb** [comment]
I would still like to see this extension available in VS 2022. Even though VS now has a built-in line spacing options, I find it not granular enough. Also the ability to compress comments is very nice.

**Comment by ghost** [comment]
> I would still like to see this extension available in VS 2022. Even though VS now has a built-in line spacing options, I find it not granular enough. Also the ability to compress comments is very nice.

You are right. The built-in solution is just adding a margin at the bottom of the line and it doesn't look nice when you are hovering a line.

I guess we still need LinePress for a better experience.

**Comment by stkb** [comment]
First a little info: I've found that with the VS2022 "line spacing" option, while you can't set it to an exact value in the options, (only 1, 1.15, 1.5 etc), you can export the TextEditor/General settings to a file, edit the value to something you want (eg 1.3), and then re-import the settings. (It shows as "1.0" in the settings dialog but does apply.) It's probably also possible to change it via the registry if you know where to look. It can take a while to experiment this way though.

Second I've found that you can combine the built-in line spacing option with the LinePress extension. Eg. I have 1.15 line spacing as a "base", and then use LinePress to stretch another 25%, but not for comments.

I've submitted PR #13 that adds support for VS 2022. If you want you can compile your own .VSIX file from that code and install it into 2022.
